### ISSUE TITLE
Settings menu doesn't fit on the screen [Linux]

### ISSUE THREAD

**Issue by Punkoivan** [rwtnb/Drumsthesia#16]
Hello! thank's for the cool app, man!

When open settings, it's not fit on my screen (please see screenshot)
![Screenshot_20230323_194659](https://user-images.githubusercontent.com/16512299/227297579-7b015356-7a3d-4f71-b2f2-f6d53da8af14.png)

I guess scaling or at least scrollbar will work.
(it's laptop's screen, on 4k screen everything is cool)

Version: pulled latest code (17:30 UTC 23.03.23) and built it.

Thanks in advance

**Comment by rwtnb** [comment]
Hi, I don't have an easy linux machine to test right now.
I'll try a VM and see if I can reproduce it this weekend.

Until then if you're willing to try to fix it yourself, it must be something around here: https://github.com/rwtnb/Drumsthesia/blob/main/src/scene/menu_scene/iced_menu/mod.rs#L403 

Must be an easy fix.

**Comment by Punkoivan** [comment]
> Hi, I don't have an easy linux machine to test right now. I'll try a VM and see if I can reproduce it this weekend.
> 
> Until then if you're willing to try to fix it yourself, it must be something around here: https://github.com/rwtnb/Drumsthesia/blob/main/src/scene/menu_scene/iced_menu/mod.rs#L403
> 
> Must be an easy fix.

Yeah, I've read iced docs and have found a solution. Please see PR.
